jol-cli/src/main/java/org/openjdk/jol/operations/ClasspathedOperation.java

```diff
@@ -67,6 +73,10 @@ public void run(String[] args) throws Exception {
             if (set.has(optClassPath)) {
                 ClassUtils.addClasspathEntries(optClassPath.values(set));
             }
+
+            if (set.has(optFactory)) {
+                factoryClass = ClassUtils.loadClass(optFactory.value(set));
```

Comment:
Can you check what happens if class cannot be loaded? I think it should also fall through to `printHelpOn` with some useful message?